File: fizzbuzz/fizzbuzz.js
var fizzBuzz = function() {
  for (var i = 0; i < 100; i++) {
    if (i % 15 === 0) {
      console.log("FizzBuzz");
    } else if (i % 3 === 0) {
      console.log("Fizz");
    } else if (i % 5 === 0) {
      console.log("Buzz");
    } else {
      console.log(i);
    }
  }
}

fizzBuzz();

// var dude = [1,2,3,4,5,6]

// for (number in dude) {
//   number.yourMOM()
// }

File: fizzbuzz/fizzbuzz.rb
def fizzbuzz(start=1, finish=100)
  output = []
  fizz = "Fizz"
  buzz = "Buzz"
  range = (start..finish).to_a

  range.each do |int|
    output << fizz+buzz if (int % 15).zero?
    output << fizz elsif (int % 3).zero?
    output << buzz elsif (int % 5).zero?
  end
end

fizzbuzz